### ISSUE TITLE
Issues with broadcasting TX through blockcyper

### ISSUE THREAD

**Issue by muneeb-ali** [blockstack/blockstore#186]
Here is the latest issue I'm running into.

```
{
    "error": "Exception: Tx hash missing from blockcypher response: {u'error': u'Error sending transaction: Rejected script for input 1 referencing 8e9b504eb5cb8c1d2a58a6f250e577fa8c1be3c649a6d9b595233cdce63e7f20 at 1..'}",
    "traceback": [
        "Traceback (most recent call last):",
        "  File \"/Users/muneeb/Dropbox (Personal)/work/blockstack/blockstore/blockstore/blockstored.py\", line 454, in blockstore_name_preorder",
        "    name_fee, testset=blockstore_opts['testset'], subsidy_public_key=public_key, tx_only=tx_only )",
        "  File \"/Users/muneeb/Dropbox (Personal)/work/blockstack/blockstore/blockstore/lib/operations/preorder.py\", line 170, in broadcast",
        "    response = serialize_sign_and_broadcast(inputs, outputs, private_key_obj, blockchain_client)",
        "  File \"/Users/muneeb/.virtualenvs/blockstore/lib/python2.7/site-packages/pybitcoin/transactions/network.py\", line 156, in serialize_sign_and_broadcast",
        "    response = broadcast_transaction(signed_tx, blockchain_client)",
        "  File \"/Users/muneeb/.virtualenvs/blockstore/lib/python2.7/site-packages/pybitcoin/transactions/network.py\", line 51, in broadcast_transaction",
        "    return blockcypher.broadcast_transaction(hex_tx, blockchain_client)",
        "  File \"/Users/muneeb/.virtualenvs/blockstore/lib/python2.7/site-packages/pybitcoin/services/blockcypher.py\", line 90, in broadcast_transaction",
        "    raise Exception(err_str)",
        "Exception: Tx hash missing from blockcypher response: {u'error': u'Error sending transaction: Rejected script for input 1 referencing 8e9b504eb5cb8c1d2a58a6f250e577fa8c1be3c649a6d9b595233cdce63e7f20 at 1..'}"
    ]
}```

**Comment by jcnelson** [comment]
This probably belongs in the `pybitcoin` issue tracker, since it is responsible for the uniformity of the `broadcast_transaction` method's return value.

**Comment by muneeb-ali** [comment]
Fixed in blockstore 0.0.9 7194dc0b01195385d4967d3d8a2f945f8ee78a24
